lib/tutorials/en_US/plugins.md

```diff
@@ -1,205 +1,166 @@
 
-Additionally, the `attributes` object may contain the key `multiple` that when set to `true` tells hapi that it is safe to register your plugin more than once in the same server.
-
-### The register method
-
-As we've seen above, the `register` method accepts three parameters, `server`, `options`, and `next`.
-
-The `options` parameter is simply whatever options the user passes to your plugin. No changes are made and the object is passed directly to your `register` method.
+```javascript
+'use strict';
 
-`next` is a method to be called when your plugin has completed whatever steps are necessary for it to be registered. This method accepts only one parameter, `err`, that should only be defined if an error occurred while registering your plugin.
+exports.plugin = {
+    pkg: require('./package.json'),
+    register: async function (server, options) {
 
-The `server` object is a reference to the `server` your plugin is being loaded in.
+        // Create a route for example
 
-#### `server.select()`
+        server.route({
+            method: 'GET',
+            path: '/test',
+            handler: function (request, h) {
 
-Servers can have connections added with a label assigned to them:
+                return 'hello, world';
+            }
+        });
 
-```javascript
-const server = new Hapi.Server();
-server.connection({ labels: ['api'] });
+        // ...
+    }
+};
 ```
 
-This label can then be used to apply plugins and other settings only to specific connections by using the `server.select()` method.
+Note that in the first example, we set the `name` and `version` properties explicitly, however in the second we set a `pkg` parameter with the contents of package.json as its value. Either method is acceptable.
 
-For example, to add a route only to connections with a label of `'api'`, you would use:
+Also note that when written as a module, the module should export the plugin under the `exports.plugin` property.
 
-```javascript
-const api = server.select('api');
+Additionally, the plugin object may contain the property `multiple` that when set to `true` tells hapi that it is safe to register your plugin more than once in the same server.
 
-api.route({
-    method: 'GET',
-    path: '/',
-    handler: function (request, reply) {
-        reply('api index');
-    }
-});
-```
+### The register method
 
-Multiple labels can be selected at the same time by passing an array of strings, this works as a logical OR statement. To accomplish a logical AND, you can chain calls to `server.select()` like so:
+As we've seen above, the `register` method accepts two parameters, `server` and `options`.
 
-```javascript
-// all servers with a label of backend OR api
-const myServers = server.select(['backend', 'api']);
+The `options` parameter is simply whatever options the user passes to your plugin. No changes are made and the object is passed directly to your `register` method.
```

Comment:
Consider mentioning `server.register()` here for context.  For example,
> ...whatever options the user passes to your plugin when registering it using `server.register()`.

Reply:
I can sort of see it both ways—passing options is covered below.  Another option is to just reference the section coming up.